src/Model/Storage/ReferenceStorageException.php

```diff
@@ -21,31 +21,11 @@
 
 namespace Friendica\Model\Storage;
```

Comment:
Not sure these exception classes should be in the `Model\` namespace, maybe we should start a new namespace for all the Friendica exceptions.

Reply:
I'd prefer having the exceptions at the place where the business logic resides. I think it's better to have explicit exceptions rather than "global" exceptions. f.e. I'm not a fan that we use the `InternalServerException` in cases of the backend errors. It's too common and should be the "Response" to a caller, not the "root-cause"

Reply:
Got it.